<?xml version="1.0" encoding="UTF-8"?>
<launch>
<!-- 
Launches all necessary nodes that transform various frames
-->
    <arg name="topic_input" default="/scan" />
    <arg name="topic_output" default="/scan/lidar/cartesian" />
    <arg name="base_link" default="ego_racecar/base_link" />
    <!-- tf_laserscan_to_pointcloud node -->
    <node
      respawn="true"
      pkg="car_tf"
      type="laserscan_transformer"
      name="$(anon laserscan_transformer)"
        output="screen" >
        <param name="topic_input" type="string" value="$(arg topic_input)" />
        <param name="topic_output" type="string" value="$(arg topic_output)" />
        <param name="base_link" type="string" value="$(arg base_link)" />
    </node>
	
</launch>
